### PR TITLE
new validation for torch

### PR DESCRIPTION
**Pull request by YilingQiao**
new validation

---
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d-ml/169)

### CODE REVIEW

File: ml3d/datasets/semantickitti.py [hunk 1]
```diff
@@ -187,11 +187,12 @@ def save_test_result(self, results, attr):
...
         test_file_name = name_points
 
-        pred = results['predict_labels']
+        pred = results['predict_labels'] + 1
+
+        np.save("pred_label.npy", pred)
 
         store_path = join(save_path, name_points + '.label')
-        for ign in cfg.ignored_label_inds:
-            pred[pred >= ign] += 1
```
Comment (sanskar107) [inline ml3d/datasets/semantickitti.py:194]: I guess, this is generalised implementation for multiple ignored indices.

File: ml3d/tf/models/network_blocks.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def block_decider(block_name, radius, in_dim, out_dim, layer_ind, cfg):
                           out_dim,
                           cfg.use_batch_norm,
                           cfg.batch_norm_momentum,
-                          l_relu=cfg.get('l_relu', 0.1))
```
Comment (sanskar107) [inline ml3d/tf/models/network_blocks.py:85]: as far as i remember, 0.2 was for converted weights from toronto3d and 0.1 was for all other datasets trained using our pipeline. Will it still work?

Comment (YilingQiao) [reply ml3d/tf/models/network_blocks.py:85]: The default slope rate is 0.2 in the original code. (https://github.com/HuguesTHOMAS/KPConv/blob/07861d8e40244c20ede5a362052e01f61dcc012a/models/network_blocks.py#L166)

I will also run the inference for all the datasets again.

Comment (YilingQiao) [reply ml3d/tf/models/network_blocks.py:85]: The default slope rate is 0.2 in the original code. (https://github.com/HuguesTHOMAS/KPConv/blob/07861d8e40244c20ede5a362052e01f61dcc012a/models/network_blocks.py#L166)

I will also run the inference for all the datasets again.

File: ml3d/datasets/semantickitti.py (hunk 1 above)
Comment (YilingQiao) [reply ml3d/datasets/semantickitti.py:194]: Done

**Review by sanskar107 — APPROVED** [review]

**Pull request merged**